client/src/components/ReviewsAndRatings/Characteristics/Comfort.jsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+
+function Comfort(props) {
+  if (props.characteristics.Comfort !== undefined) {
+    return (
+      <div>
+        <props.styles.title>Comfort</props.styles.title>
+        <props.styles.slider
```

Comment:
Where did you find this style of writing JSX tags? I want to read up more on the merits of this @Sergiojh-design

Reply:
It was actually an idea I had and it worked. I wanted to pass down my style component to all my files in "Characteristics" and only define it on my parent "Characteristics.jsx" and thought about passing my styles as props where I defined an object called styles and placed all my styling components in that object.